sdk/formrecognizer/Azure.AI.FormRecognizer/src/Azure.AI.FormRecognizer.csproj

```diff
@@ -17,11 +17,11 @@
     <Compile Include="$(AzureCoreSharedSources)Argument.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
     <Compile Include="$(AzureCoreSharedSources)ArrayBufferWriter.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
     <Compile Include="$(AzureCoreSharedSources)AzureKeyCredentialPolicy.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)AzureResourceProviderNamespaceAttribute.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
```

Comment:
Out of curiosity, what does this class do?  (the Az RP Namspace Attribute)

Reply:
We're using it in `AssemblyInfo` to set a resource provider namespace:

https://github.com/Azure/azure-sdk-for-net/blob/7c220deddf54e960b746909bd0ff659ed28e9d76/sdk/formrecognizer/Azure.AI.FormRecognizer/src/Properties/AssemblyInfo.cs#L7

Core uses it to apply an `az.namespace` attribute to all diagnostic scopes we create. Here's the error message we get from Core if we don't set it:

> System.InvalidOperationException : All diagnostic scopes should have 'az.namespace' attribute, make sure the assembly containing **ClientOptions type is marked with the AzureResourceProviderNamespace attribute specifying the appropriate provider. This attribute should be included in AssemblyInfo, and can be included by pulling in AzureResourceProviderNamespaceAttribute.cs using the AzureCoreSharedSources alias.